<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="0.0000" tests="1" failures="1">
  <testsuite name="Root Suite" timestamp="2022-07-11T17:21:48" tests="0" file="cypress/integration/uploadArquivo.spec.js" time="0.0000" failures="0">
  </testsuite>
  <testsuite name="Uplaod File" timestamp="2022-07-11T17:21:48" tests="1" time="0.0000" failures="1">
    <testcase name="Uplaod File Upload" time="0.0000" classname="Upload">
      <failure message="Timed out retrying after 4050ms: `cy.click()` failed because this element:

`&lt;a data-method=&quot;get&quot; href=&quot;/outros/uploaddearquivos&quot;&gt;Upload ...&lt;/a&gt;`

is being covered by another element:

`&lt;div class=&quot;col s3&quot;&gt;...&lt;/div&gt;`

Fix this problem, or use {force: true} to disable error checking.

https://on.cypress.io/element-cannot-be-interacted-with" type="CypressError"><![CDATA[CypressError: Timed out retrying after 4050ms: `cy.click()` failed because this element:

`<a data-method="get" href="/outros/uploaddearquivos">Upload ...</a>`

is being covered by another element:

`<div class="col s3">...</div>`

Fix this problem, or use {force: true} to disable error checking.

https://on.cypress.io/element-cannot-be-interacted-with
    at $Cy.ensureDescendents (https://automacaocombatista.herokuapp.com/__cypress/runner/cypress_runner.js:164036:78)
    at ensureDescendents (https://automacaocombatista.herokuapp.com/__cypress/runner/cypress_runner.js:150250:8)
    at ensureDescendentsAndScroll (https://automacaocombatista.herokuapp.com/__cypress/runner/cypress_runner.js:150257:14)
    at ensureElIsNotCovered (https://automacaocombatista.herokuapp.com/__cypress/runner/cypress_runner.js:150389:5)
    at runAllChecks (https://automacaocombatista.herokuapp.com/__cypress/runner/cypress_runner.js:150579:52)
    at retryActionability (https://automacaocombatista.herokuapp.com/__cypress/runner/cypress_runner.js:150603:16)
    at tryCatcher (https://automacaocombatista.herokuapp.com/__cypress/runner/cypress_runner.js:13022:23)
    at Function.Promise.attempt.Promise.try (https://automacaocombatista.herokuapp.com/__cypress/runner/cypress_runner.js:10296:29)
    at whenStable (https://automacaocombatista.herokuapp.com/__cypress/runner/cypress_runner.js:168760:63)
    at https://automacaocombatista.herokuapp.com/__cypress/runner/cypress_runner.js:168257:14
    at tryCatcher (https://automacaocombatista.herokuapp.com/__cypress/runner/cypress_runner.js:13022:23)
    at Promise._settlePromiseFromHandler (https://automacaocombatista.herokuapp.com/__cypress/runner/cypress_runner.js:10957:31)
    at Promise._settlePromise (https://automacaocombatista.herokuapp.com/__cypress/runner/cypress_runner.js:11014:18)
    at Promise._settlePromise0 (https://automacaocombatista.herokuapp.com/__cypress/runner/cypress_runner.js:11059:10)
    at Promise._settlePromises (https://automacaocombatista.herokuapp.com/__cypress/runner/cypress_runner.js:11139:18)
    at Promise._fulfill (https://automacaocombatista.herokuapp.com/__cypress/runner/cypress_runner.js:11083:18)
From Your Spec Code:
    at Context.eval (https://automacaocombatista.herokuapp.com/__cypress/tests?p=cypress\integration\uploadArquivo.spec.js:110:68)]]></failure>
    </testcase>
  </testsuite>
</testsuites>